conduit/plugins/processors/filterprocessor/gen/generate.go

```diff
@@ -59,6 +59,7 @@ var ignoreTags = map[string]bool{
 	"txn.apap": true,
 	"txn.apsu": true,
 	"dt.lg":    true,
+	"dt.sa":    true,
```

Comment:
Cannot hand this array type [EvalDelta.SharedAccounts](https://github.com/algorand/go-algorand-sdk/blob/77dbcb81b8ce3ff34144e3a37cc110cc024e4642/types/block.go#L265)